resources/migrations/000_migrations.yaml

```diff
@@ -15337,6 +15337,23 @@ databaseChangeLog:
                     nullable: false
                   defaultValue: '#31698A'
 
+  - changeSet:
+      id: v48.00-021
+      author: piranha
+      comment: 'Store metabase version in cards'
+      changes:
+        - addColumn:
+            tableName: report_card
+            columns:
+              - column:
+                  name: metabase_version
+                  type: varchar(100)
+                  remarks: 'Metabase version which made a given revision.'
+      rollback:
+        - dropColumn:
+            tableName: report_card
+            columnName: metabase_version
```

Comment:
This rollback does not need to be defined, and it's better if you leave it out to avoid small mistakes. Liquibase automatically creates rollbacks for certain operations, like `addColumn`.
example https://github.com/metabase/metabase/blob/4fbb73408784e52bef448b4ca1c2329a3a85f8cc/resources/migrations/000_migrations.yaml#L15151-L15162

Reply:
wow, cool!